dbt/models/reporting/reporting.res_report_summary.sql

```diff
@@ -157,25 +60,25 @@ aggregated_values AS (
 ),
 
 -- Aggregate and stack stats on sales for each reporting group
-all_sales AS (
+aggregated_sales AS (
```

Comment:
No logic has changed in this CTE; I've just renamed it to more closely match the naming pattern used by `aggregated_values` and its ancestors.